core/tests/protractor_utils/CommunityDashboardTranslateTextTab.js

```diff
@@ -20,21 +20,75 @@
 var waitFor = require('../protractor_utils/waitFor.js');
 
 var CommunityDashboardTranslateTextTab = function() {
-  var selectableLanguageElements = element(
+  var selectorContainer = element(
     by.css('.protractor-test-language-selector'));
-  var selectedLanguageElement = selectableLanguageElements.element(
-    by.css('option:checked'));
+  var selectedLanguageElement = selectorContainer.element(
+    by.css('.protractor-test-language-selector-selected'));
+  var dropdown = selectorContainer.element(
+    by.css('.protractor-test-language-selector-dropdown'));
+  var featuredLanguageContainer = selectorContainer.element(
+    by.css('.protractor-test-featured-language-container'));
+  var featuredLanguageElements = selectorContainer.all(
+    by.css('.protractor-test-featured-language'));
+  var featuredLanguageTooltipElements = selectorContainer.all(
+    by.css('.protractor-test-featured-language-tooltip'));
+  var featuredLanguageExplanation = selectorContainer.element(
+    by.css('.protractor-test-language-selector-featured-explanation'));
+
+  var _openSelector = async function() {
+    await waitFor.elementToBeClickable(
+      selectorContainer,
+      'Language selector taking too long to be clickable'
+    );
+    await selectorContainer.click();
+  };
 
   var _selectLanguage = async function(language) {
-    await selectableLanguageElements.element(
-      by.cssContainingText('option', language)).click();
+    await _openSelector();
+    await selectorContainer.element(
+      by.cssContainingText(
+        '.protractor-test-language-selector-option',
+        language
+      )).click();
   };
 
   this.changeLanguage = async function(language) {
     await _selectLanguage(language);
     await waitFor.pageToFullyLoad();
   };
 
+  this.mouseoverFeaturedLanguageTooltip = async function(index) {
+    await _openSelector();
+    await waitFor.visibilityOf(
+      featuredLanguageContainer,
+      'Featured languages took too long to display'
+    );
+    await browser.actions().mouseMove(
+      featuredLanguageTooltipElements.get(index)
+    ).perform();
```

Comment:
Do we need two `await` here? One for mouseMove and others for perform? (not sure!)

Reply:
No, I don't believe so